public/static/docs/api-reference/open.md

```diff
@@ -0,0 +1,188 @@
+# open()
+
+Opens a tracked file.
+
+```py
+dvc.api.open(path: str,
+             repo: str = None,
+             rev: str = None,
+             remote: str = None,
+             mode: str = "r",
+             encoding: str = None)
+```
+
+#### Usage:
+
+```py
+import dvc.api
+
+with dvc.api.open(
+        'get-started/data.xml',
+        repo='https://github.com/iterative/dataset-registry'
+        ) as fd:
+    # ... fd is a file descriptor that can be processed normally.
+```
+
+## Description
+
+Open file or model (`path`) tracked in a <abbr>DVC project</abbr> (by DVC or
+Git), and generate a corresponding
+[file object](https://docs.python.org/3/glossary.html#term-file-object).
+
+> The exact type of file object generated depends on the `mode` used. For more
+> details, please refer to Python's
+> [`open()`](https://docs.python.org/3/library/functions.html#open) built-in,
+> which is used under the hood.
+
+`dvc.api.open()` may only be used as a
+[context manager](https://www.python.org/dev/peps/pep-0343/#context-managers-in-the-standard-library)
+(using the `with` keyword, as shown in the **Examples** below).
+
+> Use `dvc.api.read()` to get the file's contents directly – no _context
```

Comment:
it's not about even directness - it's handy to get it all at once

Reply:
Yes, that's what I was trying to express. Directly vs. having to go through a `with` statement. Rephrased to "to get the complete file contents in a single function call".